.changeset/mean-berries-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/hardhat-toolbox": patch
```

Comment:
```suggestion
"@nomicfoundation/hardhat-toolbox": major
```

I think this needs to be a major, otherwise, people with the previous version will get this update and end up in a broken state. Or, npm 7+ won't even install it.

Being a major, I think we should provide a better description.